axelor-purchase/src/main/java/com/axelor/apps/purchase/web/PurchaseOrderController.java

```diff
@@ -388,6 +388,13 @@ public void validate(ActionRequest request, ActionResponse response) {
     }
   }
 
+  public void cancel(ActionRequest request, ActionResponse response) {
```

Comment:
Same issue as above, migrating existing code takes time but for new controller we should always trace all exceptions that could occur.